metatron-app/metatron-discovery
# F#2806 exploration catalog css bug

### Description

exploration css bug
**Related Issue** : 
#2806 



### How Has This Been Tested?

1. Exploration -> Data 
2. In LNB check selected catalog's bold feature works properly
3. Check folding icon toggle properly
4. Check folding icon's tooltip shows properly
5. check New or Update and My Favorite list shows properly
#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
